sys/net/gnrc/link_layer/lorawan/gnrc_lorawan.c

```diff
@@ -132,6 +132,8 @@ static void _config_radio(gnrc_lorawan_t *mac, uint32_t channel_freq, uint8_t dr
 
 static void _configure_rx_window(gnrc_lorawan_t *mac, uint32_t channel_freq, uint8_t dr)
 {
+    DEBUG("gnrc_lorawan: RX2 Frequency: %luHz \n", channel_freq);
```

Comment:
Same thing here, this is called for RX1 and RX2

Reply:
It was supposed to be RX frequency. Updated.